### PR TITLE
get the group owner guid properly

### PR DESCRIPTION
**Pull request by pandurx**
The problem with the stickies was that it was not getting the right group owner guid (in group forums, editing the forum topic)

### CODE REVIEW

File: mod/gcforums/views/default/forms/gcforums/edit.php [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <?php
 
 //elgg_push_breadcrumb('step 1');
```
Comment (nschonni) [inline mod/gcforums/views/default/forms/gcforums/edit.php]: Is this comment related to the code you inserted below? otherwise it might be confusing

Comment (pandurx) [inline mod/gcforums/views/default/forms/gcforums/edit.php]: yes, that is inserted by accident. i will have it removed.

File: mod/gcforums/views/default/forms/gcforums/edit.php [hunk 2]
```diff
@@ -32,11 +31,17 @@
...
+// only group owner/moderators/admin can do sticky topics
 $gcf_current_user_guid = elgg_get_logged_in_user_guid();
 $gcf_moderator_users_guid = array();
 
-$gcf_moderator_user[] = get_entity($vars['group_guid'])->owner_guid;
+// not getting the group owner guid properly (use library in cp_notification)
+// check if the plugin cp_notification is active before using the function get_forum_in_group
+if (elgg_is_active_plugin('cp_notifications')) {
+	elgg_load_library('elgg:gc_notification:functions');
+	$gcf_moderator_user[] = get_entity(get_forum_in_group($object->getGUID(),$object->getGUID()))->owner_guid;
```
Comment (nschonni) [inline mod/gcforums/views/default/forms/gcforums/edit.php]: This will get wiped out on L51, so maybe L45-53 need to be in and `else` branch of this conditional

Comment (nschonni) [inline mod/gcforums/views/default/forms/gcforums/edit.php]: Nevermind, I don't understand PHP arrays :wink:

Comment (pandurx) [inline mod/gcforums/views/default/forms/gcforums/edit.php]: PHP is hard to understand anyways. :p

File: mod/gcforums/views/default/forms/gcforums/edit.php [hunk 3]
```diff
@@ -32,11 +31,17 @@
...
 
-// cyu - patched that only group owner/moderators/admin can do sticky topics
+// only group owner/moderators/admin can do sticky topics
 $gcf_current_user_guid = elgg_get_logged_in_user_guid();
 $gcf_moderator_users_guid = array();
 
-$gcf_moderator_user[] = get_entity($vars['group_guid'])->owner_guid;
+// not getting the group owner guid properly (use library in cp_notification)
+// check if the plugin cp_notification is active before using the function get_forum_in_group
+if (elgg_is_active_plugin('cp_notifications')) {
```
Comment (nschonni) [inline mod/gcforums/views/default/forms/gcforums/edit.php]: Maybe add the `<suggests>` to https://github.com/gctools-outilsgc/gcconnex/blob/gcconnex/mod/gcforums/manifest.xml

Comment (pandurx) [inline mod/gcforums/views/default/forms/gcforums/edit.php]: @nschonni added suggested plugin in the manifest file. @Steph4104 fixed the notification for the forum and category creation. the subjects should be ok now.
